server/game/cards/23-AHaH/LordProtectorsRetainer.js

```diff
@@ -7,10 +7,8 @@ class LordProtectorsRetainer extends DrawCard {
         this.interrupt({
             canCancel: true,
             when: {
-                onCardAbilityInitiated: event => event.ability.targets.some(target => target.type === 'choose') &&
-                                                event.targets.hasSingleTarget() &&
+                onCardAbilityInitiated: event => event.targets.hasSingleTarget() &&
                                                 event.targets.anySelection(selection => (
-                                                    selection.choosingPlayer !== this.controller &&
                                                     selection.value.controller === this.controller &&
```

Comment:
These changes are unfortunately incorrect. This ability _**should**_ only trigger on cards which use the word "choose" (thus that first check), and should only work when an opponent is choosing.

Additionally, the issue lies within the fact that `event.targets` is incorrect, where it should be `event.ability.targets`. This adjustment *should* fix the bug; I'd revert the rest :)

Reply:
It seems I missed the additional line that was removed, being the one containing `event.ability.targets.some(target => target.type === 'choose') &&`. This check needs to be there

Reply:
the text of the card is this: Interrupt: When the effects of an ability that chooses a Lord or Lady character you control as its only target would initiate, return Lord Protector's Retainer to your hand to cancel those effects. (Max once per phase.)

I would assume you should be able to trigger it even if you were the one choosing (perhaps redirected with (High Septon, etc) - thats for the deleted check for opponent.

As for the check for targetingType "choose" - that seems to be within the hasSingleTarget() method and hence is redundant.

Reply:
Fair enough with that; this will still need the `event.ability.targets` adjustment as that's breaking it, but the rest seems good :)

Reply:
I tested the latest iteration, and it does not work unfortunately.

I did also notice however that it does not have `event.ability.targets`, and is rather using `event.targets`. It's important to note that High Septon's interrupt is slightly different to Lord Protector's Retainer.

High Septon reacts to when the **target is chosen**; this is why the event is `onTargetsChosen`, and has `event.targets` populated.
Lord Protector's Retainer is when the **ability is initiating**; this is why the event is `onCardAbilityInitiated`, and does not have `event.targets` populated, but rather has that information within `events.ability.targets`. LPR is closer to `Fear Cuts Deeper Than Swords`, rather than High Septon. :)

I am certain that we simply need to change `events.targets` to `events.ability.targets`; I have also tested the adjustment and it fixes the issue.